package.json

```diff
@@ -103,10 +98,11 @@
     "terser": "^4.0.0",
     "ts-jest": "^24.0.2",
     "ts-json-schema-generator": "^0.44.0",
-    "typescript": "^3.4.5",
```

Comment:
Why do you fix the ts version?

Reply:
Because 3.5 does break the code, so it shouldn't be ^, I guess ~ is fine.

Reply:
Let's do ~ then